app/helpers/application_helper.rb

```diff
@@ -1392,12 +1392,4 @@ def remote_function(options)
   def appliance_name
     MiqServer.my_server.name
   end
-
-  def rbac_common_feature_for_buttons(pressed)
```

Comment:
@jzigmund intent to add this method was that we can keep all button ids in one place that use some kind of common feature for role_allows calls, with that method removed those will be all over the place in the if/else checks.